lib/util/utils.ts

```diff
@@ -860,3 +910,33 @@ export const shuffleArray = (a: any[]) => {
 export const printWarning = (...args: string[]) => {
   console.log("\x1b[33m%s\x1b[0m", ...args);
 };
+
+// eslint-disable-next-line prefer-const
+export let usePsudorandom = {
+  flag: false,
+  seed: 0,
+};
+
+function* mulberry32(seed: number) {
```

Comment:
what does this function do? can you add some comment

Reply:
this is to generate random string. I'll add comments